src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = false;
```

Comment:
The 'hidden' property should belong to the Herbivore class, not the Animal class, as it's specific to Herbivores. Move this property to the Herbivore class constructor.